cli/src/tests.rs

```diff
@@ -875,7 +875,7 @@ async fn remove_device(tmp_path: TmpPath) {
 
     let alice_device_file = tmp_path
         .join("config")
-        .join("parsec-v3-alpha")
+        .join("parsec3/libparsec")
```

Comment:
Probably doesn't change much but you could also separate those:
```
.join("config")
.join("parsec3")
.join("libparsec")
...
```

Reply:
Just do everything on one `join` expect for that last value which is dynamic